README.md

```diff
@@ -61,7 +61,9 @@ When an issuer.example context wants to provide tokens to a user (i.e. when the
 ```
 fetch('<issuer>/.well-known/trust-token', {
   trustToken: {
-    type: 'token-request',
+    type: '0xFF53',
```

Comment:
I think this actually wants to be type: 'trust-token', with the token type being kept in crypto side (key commitments and such).